--- qttools-everywhere-src-5.10.0-beta/src/linguist/lrelease/lrelease.pro.0001~	2017-10-10 23:51:58.286310553 +0200
+++ qttools-everywhere-src-5.10.0-beta/src/linguist/lrelease/lrelease.pro	2017-10-10 23:55:00.514585084 +0200
@@ -13,3 +13,7 @@ QT_TOOL_ENV += qmake
 
 QMAKE_TARGET_DESCRIPTION = "Qt Translation File Compiler"
 load(qt_tool)
+
+# When linking with libQt5Bootstrap.a we need zlib too
+LIBS += -Wl,--no-as-needed
+LIBS += -lz
