diff -up qttools-everywhere-src-5.11.0-rc2/mkspecs/features/qt_find_clang.prf.0003~ qttools-everywhere-src-5.11.0-rc2/mkspecs/features/qt_find_clang.prf
--- qttools-everywhere-src-5.11.0-rc2/mkspecs/features/qt_find_clang.prf.0003~	2018-05-19 04:26:52.837066165 +0200
+++ qttools-everywhere-src-5.11.0-rc2/mkspecs/features/qt_find_clang.prf	2018-05-19 04:27:22.035374613 +0200
@@ -132,7 +132,7 @@ for(_, $$list(_)) { # just a way to brea
                         -lclangRewriteFrontend \
                         -lclangSema \
                         -lclangSerialization \
-                        -lclang_static \
+                        -lclang \
                         -lclangStaticAnalyzerCheckers \
                         -lclangStaticAnalyzerCore \
                         -lclangStaticAnalyzerFrontend \
