diff -up qttools-everywhere-src-5.15.0-rc/src/assistant/assistant/helpviewer.cpp.omv~ qttools-everywhere-src-5.15.0-rc/src/assistant/assistant/helpviewer.cpp
diff -up qttools-everywhere-src-5.15.0-rc/src/assistant/assistant/helpviewer.h.omv~ qttools-everywhere-src-5.15.0-rc/src/assistant/assistant/helpviewer.h
--- qttools-everywhere-src-5.15.0-rc/src/assistant/assistant/helpviewer.h.omv~	2020-05-06 17:23:40.192591716 +0200
+++ qttools-everywhere-src-5.15.0-rc/src/assistant/assistant/helpviewer.h	2020-05-06 17:23:53.396601066 +0200
@@ -130,7 +130,7 @@ protected:
     void wheelEvent(QWheelEvent *event) override;
     void mousePressEvent(QMouseEvent *event) override;
     void mouseReleaseEvent(QMouseEvent *event) override;
-    void resizeEvent(QResizeEvent *e) override;
+//    void resizeEvent(QResizeEvent *e) override;
 
 private slots:
     void actionChanged();
