diff -up qttools-everywhere-src-5.15.0-alpha/src/qdoc/configure.pri.4~ qttools-everywhere-src-5.15.0-alpha/src/qdoc/configure.pri
--- qttools-everywhere-src-5.15.0-alpha/src/qdoc/configure.pri.4~	2020-02-17 23:35:05.459143043 +0100
+++ qttools-everywhere-src-5.15.0-alpha/src/qdoc/configure.pri	2020-02-17 23:35:05.460143043 +0100
@@ -57,6 +57,7 @@ defineTest(qtConfTest_libclang) {
     # Assume libclang is installed on the target system
     isEmpty(LLVM_INSTALL_DIR) {
         llvmConfigCandidates = \
+            llvm-config-10 \
             llvm-config-9 \
             llvm-config-8 \
             llvm-config-7 \
