diff -up qttools-everywhere-src-5.11.0-rc/mkspecs/features/qt_find_clang.prf.omv~ qttools-everywhere-src-5.11.0-rc/mkspecs/features/qt_find_clang.prf
--- qttools-everywhere-src-5.11.0-rc/mkspecs/features/qt_find_clang.prf.omv~	2018-05-10 01:43:35.362954056 +0200
+++ qttools-everywhere-src-5.11.0-rc/mkspecs/features/qt_find_clang.prf	2018-05-10 01:43:41.584778539 +0200
@@ -122,7 +122,7 @@ for(_, $$list(_)) { # just a way to brea
                       -lclangRewriteFrontend \
                       -lclangSema \
                       -lclangSerialization \
-                      -lclang_static \
+                      -lclang \
                       -lclangStaticAnalyzerCheckers \
                       -lclangStaticAnalyzerCore \
                       -lclangStaticAnalyzerFrontend \
